addons/spreadsheet/static/src/o_spreadsheet/migration.js

```diff
@@ -34,6 +34,24 @@ migrationStepRegistry.add("18.1.2", {
     },
 });
 
+migrationStepRegistry.add("18.3.1", {
+    migrate(data) {
+        for (let sheet of data.sheets || []) {
+            for (let figure of sheet.figures || []) {
+                if (
+                    figure.tag === "chart" &&
+                    figure.data.type.startsWith("odoo_")
```

Comment:
shouldn't we be more precise ? `odoo_pie`, `odoo_geo` or `odoo_radar` don't give a damn about cumulated start.

Or is it something set to every type of chart, that way it works when changing the type to a line/bar ?

Reply:
👍  indeed we don't need it at all